x/ibc/03-connection/keeper/handshake_test.go

```diff
@@ -75,43 +77,103 @@ func (suite *KeeperTestSuite) TestConnOpenTry() {
 			clientA, clientB = suite.coordinator.SetupClients(suite.chainA, suite.chainB, clientexported.Tendermint)
 			_, _, err := suite.coordinator.ConnOpenInit(suite.chainA, suite.chainB, clientA, clientB)
 			suite.Require().NoError(err)
+
+			// retrieve client state of chainA to pass as counterpartyClient
+			counterpartyClient, found = suite.chainA.App.IBCKeeper.ClientKeeper.GetClientState(suite.chainA.GetContext(), clientA)
```

Comment:
`suite.chainA.GetClientState(clientA)` ? think the testing pkg helper func does these two lines